coops-ui/package.json

```diff
@@ -13,6 +13,8 @@
   },
   "dependencies": {
     "@types/vuelidate": "^0.7.4",
+    "@typescript-eslint/eslint-plugin": "^2.3.0",
+    "@typescript-eslint/parser": "^2.3.0",
```

Comment:
Are these prod or dev dependencies?

Reply:
Oh good question. They might just be dev dependencies.